scalafix-core/shared/src/main/scala/scalafix/rewrite/RewriteCtx.scala

```diff
@@ -1,20 +1,27 @@
 package scalafix
 package rewrite
-import scala.meta.Tree
+import scala.meta._
 import scala.meta.contrib.AssociatedComments
 import scala.meta.inputs.Input
 import scala.meta.io.AbsolutePath
 import scala.meta.tokens.Tokens
 import scalafix.syntax._
 import scalafix.config.ScalafixConfig
 import scalafix.config.ScalafixReporter
+import scalafix.patch.PatchOps
+import scalafix.patch.TokenPatch
+import scalafix.patch.TokenPatch.Add
+import scalafix.patch.TreePatch
+import scalafix.patch.TreePatch._
 import scalafix.util.MatchingParens
 import scalafix.util.TokenList
+import org.scalameta.FileLine
 import org.scalameta.logger
 
 /** Bundle of useful things when implementing [[Rewrite]]. */
-case class RewriteCtx(tree: Tree, config: ScalafixConfig) {
-  def syntax =
+case class RewriteCtx(tree: Tree, config: ScalafixConfig) extends PatchOps {
+  ctx =>
```

Comment:
Yes, but I like the convention of calling it `ctx`. The ctx is not much used to implement the patches at the moment but I suspect that later we may need it to support configuration etc.